promql/parser/printer.go

```diff
@@ -119,11 +119,21 @@ func (node *MatrixSelector) String() string {
 	if vecSelector.Offset != time.Duration(0) {
 		offset = fmt.Sprintf(" offset %s", model.Duration(vecSelector.Offset))
 	}
+	at := ""
+	if vecSelector.Timestamp != 0 {
+		at = fmt.Sprintf(" @ %f", vecSelector.Timestamp)
```

Comment:
Is %f really the right thing to use here?

Reply:
Maybe upto 3 decimal places since we use milliseconds